src/index.js

```diff
@@ -185,6 +193,11 @@ const INNER_TEXTS = {
     endButtonElement: "End",
     headerTitleElement: "New Contact",
     plusButtonElement: "+",
+    editButtonElement: "Edit",
```

Comment:
Again, add some separators, so when list grows you will know what belogs to what.